### PR TITLE
feat: `rune create` command

### PR DESCRIPTION
**Pull request by oblador**
Screenshot 2023-05-22 at 12 23 12

### CODE REVIEW

**Review by bfelbo — COMMENTED** [review]
Awesome to add this 🥳

File: packages/rune-games-cli/src/flows/Create.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+function copyDir(srcDir: string, destDir: string) {
+  fs.mkdirSync(destDir, { recursive: true })
+
+  for (const file of fs.readdirSync(srcDir)) {
+    const srcFile = path.resolve(srcDir, file)
+    const destFile = path.resolve(destDir, file)
+    copy(srcFile, destFile)
+  }
+}
```
Comment (helios1138) [inline packages/rune-games-cli/src/flows/Create.tsx:48]: I think you can just use `fs.cpSync()` with the `recursive` option

Comment (oblador) [reply packages/rune-games-cli/src/flows/Create.tsx:48]: Docs marks it as `Experimental`

Comment (helios1138) [reply packages/rune-games-cli/src/flows/Create.tsx:48]: sounds good

**Review by helios1138 — APPROVED** [review]

**Pull request merged**